pyproject.toml

```
@@ -42,7 +42,7 @@ defusedxml = "*"
 importlib-metadata = {version = ">=1.7.0", python = "<3.8"}
 importlib-resources = {python = "<3.9", version = "*"}
 paste = {optional = true, version = "*"}
-pyopenssl = "<24.3.0"
+pyopenssl = "<24.4.0"
```

I'd bump to the newest version `25.1.0`.
```suggestion
pyopenssl = "<25.2.0"
```